docs/changes.rst

```diff
@@ -154,6 +154,121 @@ Compatibility notes:
   Since the "trick" does not work if BORG_PASSPHRASE is set, this does generally
   not affect scripts.
 
+New features:
+
+- integrity checking for important files used by borg:
+
+  - repository: index and hints files
+  - cache: chunks and files caches, archive.chunks.d
+  - temporary metadata cache contents
+- Verify most operations against securitymanager, #2487
+- use assert_secure for all commands that use the manifest
+- implement storage quotas, #2517
+- serve: add --restrict-to-repository, #2589
+- BORG_PASSCOMMAND: use external tool providing the key passphrase, #2573
+- borg export-tar, #2519
+- list: --json-lines for archive contents, #2439
+- add --debug-profile option (and also "borg debug convert-profile"), #2473
+
+Fixes:
+
+- hashindex: hash indices >2 GiB, better error reporting, #2496
```

Comment:
oh, i thought all are affected. then this was maybe less bad than i thought...

and this also means we can reconsider the backport to 1.0.

how about 32bit with PAE, it can do >> 4GB memory?